mobile/test/objective-c/EnvoyTestServer.h

```diff
@@ -4,9 +4,9 @@
 
 // Interface for starting and managing a test server. Calls into to test_server.cc
 //
-// NB: Any test that utilizes this class must have a `no-remote-exec` tag in its BUILD target.
-// EnvoyTestServer binds to a listening socket on the machine it runs on, and on CI, this
-// operation is not permitted in remote execution environments.
+// NB: Any test that utilizes this class must have a `"sandboxNetwork": "standard"`
+// `exec_properties` in its BUILD target. EnvoyTestServer binds to a listening socket on the machine
+// it runs on, and on CI, this operation is not permitted in remote execution environments.
```

Comment:
I guess this comment needs to be revised a bit more since we can bind to a listening socket as long as it's a sandboxed network?

Reply:
Done.